ui/css/treeherder-test-view.css

```diff
@@ -0,0 +1,203 @@
+@import url('https://fonts.googleapis.com/css?family=Anonymous+Pro:700|Fira+Sans+Extra+Condensed:400,500,600,700');
```

Comment:
I don't know that we need that font anyway, per se.  Eli may disagree.  But I found another narrow font that looks close enough.  So I just removed that import.

I suppose I would tend toward providing the fonts ourselves rather than relying on an outside source (even if Google is probably pretty stable).  :)